src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1543,6 +1543,45 @@ std::map<primitive_type_id, bool> layout_optimizer::get_all_onednn_impls_optimiz
     return _optimization_attributes.onednn_impls;
 }
 
+void layout_optimizer::save(cldnn::BinaryOutputBuffer& ob) const {
+    ob << _optimization_attributes.group_convolution;
+    ob << _optimization_attributes.bfyx_only_layer;
+    ob << _optimization_attributes.fs_b_yx_fsv32_network;
+    ob << _optimization_attributes.b_fs_zyx_fsv32_network;
+    ob << _optimization_attributes.b_fs_yx_fsv16_network;
+    ob << _optimization_attributes.b_fs_zyx_fsv16_network;
+    ob << _optimization_attributes.bs_fs_yx_bsv16_fsv16_network;
+
+    ob << _optimization_attributes.onednn_impls.size();
+    for (const auto& onednn_impl : _optimization_attributes.onednn_impls) {
+        ob << prim_map_storage::instance().get_type_string(onednn_impl.first);
+        ob << onednn_impl.second;
+    }
```

Comment:
I think we can do this simply.
```suggestion
    ob << _optimization_attributes.onednn_impls;
```

In cases, we could need to include this.
```c++
#include "intel_gpu/graph/serialization/map_serializer.hpp"
```